torch/linalg/__init__.py

```diff
@@ -2042,3 +2042,25 @@
     >>> torch.dist(Q.transpose(-2, -1) @ Q, torch.eye(4))
     tensor(6.2158e-07)
 """)
+
+trace = _add_docstr(_linalg.linalg_trace, r"""
+trace(input, offset=0) -> Tensor
+
+Returns the sum of the elements of the diagonal. If `offset` is given, the returned value is the sum
+of `input[i, i + offset]` for all `i`.
+
+Args:
+    input (Tensor): tensor from which the diagonals are taken.
+    offset (int): offset of the diagonal from the main diagonal. A positive value indicates
+                  an upper diagonal, and negative value a lower diagonal. Default: `0`.
+
+Example::
+
+    >>> x = torch.arange(1., 10.).view(3, 3)
```

Comment:
When working with a single tensor prefer `t` for the name, when working with multiple prefer a, b, c... when the tensors don't already have natural names.